test/extended/openstack/utils.go

```diff
@@ -216,6 +217,57 @@ func getNetworkType(ctx context.Context, oc *exutil.CLI) (string, error) {
 	return networkType, nil
 }
 
+// Check the cluster networks and returns true if it finds one ipv4 and one ipv6 network there.
+func isDualStackCluster(ctx context.Context, oc *exutil.CLI) (bool, error) {
+	ipv4Found := false
+	ipv6Found := false
+
+	networks, err := oc.AdminConfigClient().ConfigV1().Networks().Get(ctx, "cluster", metav1.GetOptions{})
```

Comment:
Instead of fetching the network info twice, here and at line 255, it's better to only fetch it once and pass it as parameter.  Maybe at line 249.